queries/python/indents.scm

```diff
@@ -38,8 +38,19 @@
 
 (ERROR
   "try"
-  ":" @indent.begin
-  (#set! indent.immediate 1))
+  .
+  ":"
+  (#set! indent.immediate 1)) @indent.begin
+
+(ERROR
+  "try"
+  .
+  ":"
+  (ERROR
+    (block
+      (expression_statement
+        (identifier) @_except) @indent.branch))
+  (#eq? @_except "except"))
```

Comment:
After trying a bit, this covers both cases:

```suggestion
(
  "try"
  .
  ":"
  .
  (ERROR
    (block
      (expression_statement
        (identifier) @_except) @indent.branch))
  (#eq? @_except "except"))
```

Reply:
And if you decided to apply this suggestion, remember to add some more tests. Even the current one looks/is good to me